src/Nethermind/Nethermind.Optimism/Rpc/RegisterOptimismRpcModules.cs

```diff
@@ -52,7 +52,7 @@ protected override void RegisterEthRpcModule(IRpcModuleProvider rpcModuleProvide
 
         if (_config.SequencerUrl is null && _logger.IsWarn)
         {
-            _logger.Warn($"SequencerUrl is not set.");
+            _logger.Warn($"SequencerUrl is not set. Nethermind will behave as a Sequencer");
```

Comment:
Personally I don't like this "defaulting" behavior and I'd rather have an explicit flag that makes Nethermind work as a sequencer since this is quite an specific use case.

Reply:
Added a `SequencerMode` flag to explicitly request Nethermind to work as a sequencer

Reply:
After discussing it with @jmederosalvarado we decided to *not* include a sequencer flag since this behavior is controlled by the CL and instead rely on the existing defaulting mechanism.